### PR TITLE
Feature/jjob

### PR DESCRIPTION
**Pull request by jkuang-exploro**
add hera.yaml
moderate clean up

### CODE REVIEW

File: workflow/defaults/places.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@
 ##--------------------------------------------------------------------------------
 #
 default_places: &default_places
+<<<<<<< HEAD
```
Comment (guillaumevernieres) [inline workflow/defaults/places.yaml:9]: @JianKuang-NOAA , please remove merge messages.

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
@JianKuang-NOAA , you forgot to add the hera yaml file.

**Comment by jkuang-exploro** [comment]
I've completed the requested changes. Please check again. Thanks

**Review by guillaumevernieres — APPROVED** [review]
looks good.

**Pull request merged**